src/streamlink/stream/hls_playlist.py

```diff
@@ -62,6 +59,12 @@ def __init__(self):
 
 
 class M3U8Parser(object):
+    _extinf_re = re.compile("(?P<duration>\d+(\.\d+)?)(,(?P<title>.+))?")
+    _attr_re = re.compile(r"([A-Z\-]+)=(\d+\.\d+|0x[0-9A-z]+|\d+x\d+|\d+|"
+                   r"\"(.+?)\"|[0-9A-z\-]+)")
+    _range_re = re.compile("(?P<range>\d+)(@(?P<offset>.+))?")
```

Comment:
`W605 invalid escape sequence`

Reply:
Needs to be `r".."` also.

Reply:
It's no from me, but what's wrong, the regex works?

Reply:
change
`re.compile("`
to
`re.compile(r"`

see https://lintlyci.github.io/Flake8Rules/rules/W605.html

Reply:
In future versions of Python it might generate an Exception - they keep deprecating it. I imagine in Python 4.0 it will not work.